code-challenges.test.js

```diff
@@ -49,4 +140,43 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
-// b) Create the function that makes the test pass.
+
+// a) Create a test with expect statements using the variable provided.
+describe("fullHouse", ()=> {
+    it("takes in an array of 5 numbers and determines whether or not the array is a full house. A full house is exactly one pair and one three of a kind.",()=>{
+        
+        const hand1 = [5, 5, 5, 3, 3]
+        const hand2 = [5, 5, 3, 3, 4]
+        const hand3 = [5, 5, 5, 5, 4]
+        const hand4 = [7, 2, 7, 2, 7]
+        expect(fullHouse(hand1)).toEqual("true")
+        expect(fullHouse(hand2)).toEqual("false")
+        expect(fullHouse(hand3)).toEqual("false")
+        expect(fullHouse(hand4)).toEqual("true")
+    
+    })
+ })
+ // b) Create the function that makes the test pass.
+ // I'm confused on how to make this work, I know I have to use if/ else statements but the numbers parts elude me.
+
+ const fullHouse('hand1')=> {
+    for (let i = 0; i < 5; i++) {
+        if fullHouse('hand1') === {
+            return true
+    }
```

Comment:
I see your pseudocode and that you are confused on how to get started, so if you need help, please reach out so we can help you however we can. For this problem, you are given an array with 5 numbers. You need to determine whether the 5 numbers include a pair and a three of a kind or not. To do this, one approach would be to make a tracker of how many times each number appears in the array, and if a number appears twice and another number appears 3 times, you know you have a full house. Please resubmit an attempt for this problem.